testers.json

```
@@ -16,5 +16,10 @@
   },
   {
     "name": "Vinodh Ilangovan"
-  }
+  },
+  {
+    "name": "Neha Moopen"
+  },
+  {
+    "name": "Lisa Hehnke"
 ]
```

The JSON is incorrect, but this fixes it.

```suggestion
  {
    "name": "Lisa Hehnke"
  }
]
```